contrib/grafana/synapse.json

```diff
@@ -56,6 +56,17 @@
         "name": "Annotations & Alerts",
         "showIn": 0,
         "type": "dashboard"
+      },
```

Comment:
The diff doesn't seem that outrageous. Some miscellaneous changes with the `x/y` stuff and looks like a graph that wasn't checked in before but overall seems reasonable.

 To get the Grafana dashboard JSON to copy here, I used the **Share** icon at the top -> **Export** -> check the **Export for sharing externally** option -> **View JSON** or **Save to file**

Thanks for the shout on this @reivilibre!